prime-router/src/main/kotlin/Translator.kt

```diff
@@ -139,13 +138,9 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
         var transformed = toReport
         if (receiver.deidentify)
             transformed = transformed.deidentify()
-        return transformed.copy(destination = receiver, bodyFormat = receiver.format)
-    }
-
-    fun buildEmptyReport(receiver: Receiver, from: Report): Report {
-        val toSchema = metadata.findSchema(receiver.schemaName)
-            ?: error("${receiver.schemaName} schema is missing from catalog")
-        return Report(toSchema, emptyList(), listOf(ReportSource(from.id, "mapping")), metadata = metadata)
+        var copy = transformed.copy(destination = receiver, bodyFormat = receiver.format)
+        copy.filteredItems.addAll(qualityFilteredReport.filteredItems)
```

Comment:
might not need this anymore

Reply:
This is required because `transformed` / `toReport` is a result of a call of `applyMapping` on `qualityFilteredReport`. which doesn't and maybe shouldn't copy the filteredItems